### PR TITLE
NCAR's GeoCAT-F2Py ready to be brought into conda-forge

### PR DESCRIPTION
**Pull request by pilotchute**
Signed-off-by: pilotchute <akootz@ucar.edu>

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
  - This a computational package from the U.S. National Center for Atmospheric Research. ~700 downloads per monthly release. 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
  - Apache 2.0 Included.
- [x] Source is from official source.
  - Source developed and maintained by NCAR staff.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
  - Does not vendor other packages.
- [x] If static libraries are linked in, the license of the static library is packaged.
  - Static libraries are packaged
- [x] Build number is 0.
  - Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
  - github version tarball used
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
  - Yes, me. Hi, how are you doing today?
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.
  - I'm always either trouble or in trouble, not gonna stop for you ;)

### CODE REVIEW

**Comment by pilotchute** [comment]
Conda-forge, The tests are passing, and thus I'm requesting the merge per the instructions. @conda-forge/staged-recipes

**Comment by pilotchute** [comment]
@conda-forge-admin, please ping conda-forge/staged-recipes

**Comment by ocefpaf** [comment]
You are pulling other commits here. Do you mind rebasing against latest main? Or maybe it is easier to send a fresh PR from an updated fork?

**Comment by pilotchute** [comment]
@ocefpaf not sure how those other commits got pulled in. So I reset to upstream main, and readded our files. Should be good to go now.

File: recipes/geocat-f2py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - python                              # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools <60.0.0                  # [build_platform != target_platform]
+    - numpy >=1.22                        # [build_platform != target_platform]
+    - {{ compiler('fortran') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - python
+    - pip
+    - setuptools <60.0.0
```
Comment (ocefpaf) [inline recipes/geocat-f2py/meta.yaml:30]: Just curios but why you cannot use newer setuptools here?

File: recipes/geocat-f2py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://github.com/NCAR/geocat-f2py/archive/v{{ version }}.tar.gz # For building from a specific tag's tarball
+  sha256: 8ee37f26084d2a2454dfd9ba476138f3d05ac9511e1d11e19ad04862e70a7fed
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: march # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/geocat-f2py.git
```
[the remark this answers is not in the document]
Comment (pilotchute) [reply recipes/geocat-f2py/meta.yaml:13]: those are for compatibility with our CI system, we are attempting to use the same meta.yaml, for both systems.

**Pull request merged**